packages/components/data-table/src/data-table.tsx

```diff
@@ -20,6 +21,7 @@ import HeaderCell from './header-cell';
 import DataRow from './data-row';
 import useManualColumnResizing from './use-manual-column-resizing-reducer';
 import ColumnResizingContext from './column-resizing-context';
+import { DataTableManagerContext } from '@commercetools-uikit/data-table-manager/src/data-table-manager';
```

Comment:
This is meant to just be a root import from data-table-manager but I am yet to find out why it would not return column data when I import that way.
```suggestion
import { DataTableManagerContext } from '@commercetools-uikit/data-table-manager';
```